parsl/dataflow/dflow.py

```diff
@@ -94,12 +96,26 @@ def __init__(self, config=None, executors=None, lazyErrors=True, appCache=True,
             self.lazy_fail = self._config["globals"].get("lazyErrors", lazyErrors)
             self.fail_retries = self._config["globals"].get("retries", retries)
             self.flowcontrol = FlowControl(self, self._config)
+            self.checkpoint_method = self._config["globals"].get("checkpointMode",
```

Comment:
@yadudoc where we have properties can they always mirror the option? That way we don't have to keep track of the mapping?

Reply:
In other words `checkpoint_mode` instead of `checkpoint_method`